ui/pages/AnvilWorkspace/LoadWorkspaceData.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { Message, Segment } from 'semantic-ui-react'
 
-import LoadWorkspaceDataForm from './components/LoadWorkspaceDataForm'
+import LoadWorkspaceDataForm from 'shared/components/form/anvil/LoadWorkspaceDataForm'
```

Comment:
the `shared/components/form` is for generic form helpers, not page-specific forms. This component belongs in `shared/components/panel/LoadWorkspaceDataForm`